includes/utils.php

```diff
@@ -159,6 +159,20 @@ function is_dt_debug() {
  * @param array $meta Array of meta as key => value
  */
 function set_meta( $post_id, $meta ) {
+	/**
+	 * Fires before Distributor sets post meta.
+	 *
+	 * Note: All sent meta is included in the `$meta` array, including excluded keys.
+	 * Take care to continue to filter out excluded keys in any further meta setting.
+	 *
+	 * @since 2.0.1
```

Comment:
```suggestion
	 * All sent meta is included in the `$meta` array, including excluded keys.
	 * Any excluded keys returned in this filter will be subsequently removed
	 * from the saved meta data.
	 *
	 * @since 2.0.0
```